File: RighpointLabs.Pourcast.Orchestrator/Models/Beer.cs
namespace RighpointLabs.Pourcast.Orchestrator.Models
{
    public class Beer
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public Brewery Brewer { get; set; }
        public double ABV { get; set; }
        public int BAScore { get; set; }
        public int RPScore { get; set; }
        public string Style { get; set; }
        public string Color { get; set; }
        public string Glass { get; set; }
        public string Slug { get; set; } 
    }
}

File: RightpointLabs.Pourcast.Web/Scripts/app/model/beer.js
define(['ko'], function(ko) {
    function Beer(beerJSON, brewery, style) {
        var self = this;

        self.id = ko.observable(beerJSON.id);
        self.name = ko.observable(beerJSON.name);
        self.abv = ko.observable(beerJSON.abv);
        self.style = ko.observable(style);
        self.brewery = ko.observable(brewery);
    };

    return Beer;
});